examples/data/quiz_questions.csv

```diff
@@ -0,0 +1,11 @@
+assessment_id,question_number,question_id
+0,5,1a7b77c1-e9e1-4f22-ba84-56a74b733419
+0,4,73cf891e-1441-433c-a00e-45fc723b0996
+0,3,8a5008a7-970b-411b-9ed7-d1970b2bed56
+0,2,9368abf2-5a6c-4254-ab57-3edfdb338511
+0,1,574085b1-06f0-485b-9fe6-a784ba37e981
+1,5,0c6d9bee-7e89-4c87-8674-da0d56090b57
+1,4,2ffd03b0-67af-4e96-9476-5d1b621f1725
+1,3,99134f44-0590-4fd3-86af-c926785b4a32
+1,2,da21613b-5d05-48d2-a07b-2bec166e5e54
+1,1,6e6df72f-a6a3-424a-9ede-38b65720d2aa
```

Comment:
I believe this might be problematic in that the `0` value won't be able to join against anything in the `assessments.csv` in the sample data as it starts with id `1`.